lang/semgrep-grammars/src/semgrep-c-sharp-pro/grammar.js

```diff
@@ -0,0 +1,144 @@
+/*
+ * semgrep-csharp
+ *
+ * Extend the original tree-sitter C# grammar with semgrep-specific constructs
+ * used to represent semgrep patterns.
+ *
+ * The language is renamed from c-sharp to csharp to match language name
```

Comment:
Looks like it's actually named `c_sharp_pro` -- is this a copy/paste error?

Reply:
yes